fluent-mechanical/wf_fm_02_mechanical.py

```diff
@@ -0,0 +1,947 @@
+
+# Export temperature image
+
+Tree.Activate([Temp])
+ExtAPI.Graphics.ViewOptions.ResultPreference.ExtraModelDisplay = (
+    Ansys.Mechanical.DataModel.MechanicalEnums.Graphics.ExtraModelDisplay.NoWireframe
+)
+ExtAPI.Graphics.ExportImage(
+    os.path.join(project_directory, "temperature.png"), image_export_format, settings_720p
+)
+"""
+)
+mechanical.download(files=os.path.join(project_directory, "temperature.png"), target_dir=OUTPUT_DIR)
+if GRAPHICS_BOOL:
+    display_image("temperature.png")
+
+
+###############################################################################
+# Setup Structural Analysis
+# -------------------------
+#
+mechanical.run_python_script(
+    """
+Model.AddStaticStructuralAnalysis()
+
+# Define analysis settings
+
+# Setup static structural analysis settings
+STAT_STRUC = Model.Analyses[1]
+STAT_STRUC_SOLN = STAT_STRUC.Solution
+STAT_STRUC_ANA_SETTING = STAT_STRUC.Children[0]
+
+STAT_STRUC_ANA_SETTING.NumberOfSteps = 1
+STAT_STRUC_ANA_SETTING.SetStepEndTime(1, Quantity('720[s]'))
+STAT_STRUC_ANA_SETTING.NumberOfSteps = 14
+
+analysis_step = (
+    (1, Quantity('1e-3[s]')),
+    (2, Quantity('2e-3[s]')),
+    (3, Quantity('20[s]')),
+    (4, Quantity('30[s]')),
+    (5, Quantity('320[s]')),
+    (6, Quantity('330[s]')),
+    (7, Quantity('350[s]')),
+    (8, Quantity('360[s]')),
+    (9, Quantity('380[s]')),
+    (10, Quantity('390[s]')),
+    (11, Quantity('680[s]')),
+    (12, Quantity('690[s]')),
+    (13, Quantity('710[s]')),
+    (14, Quantity('720[s]'))
+)
+for i, q in analysis_step:
+    STAT_STRUC_ANA_SETTING.SetStepEndTime(i,q)
+STAT_STRUC_ANA_SETTING.Activate()
+
+
+# Add Imported Body Temperature load from Transient Thermal Run
+
+STAT_STRUC.ImportLoad(Model.Analyses[0])
+imported_load = DataModel.GetObjectsByName("Imported Body Temperature")[0]
+
+table = imported_load.GetTableByName("Source Time")
+numofsteps = 14
+nCol = 2
+Ana_time = ["1e-3","2e-3","20","30","320","330","350","360","380","390","680","690","710","720"]
+
+for i in range(numofsteps-1):
+    table.Add(None)
+
+for i in range(numofsteps):
+    for j in range(nCol):
+        table[i][j] = Ana_time[i]
+
+imported_load.ImportLoad()
+
+# Apply Fixed Support Condition
+
+Fixed_Support = STAT_STRUC.AddFixedSupport()
+selection = NS_GRP.Children[3]
+Fixed_Support.Location = selection
+"""
+)
+
+###############################################################################
+# Sovle and post-process the results
+# ----------------------------------
+#
+mechanical.run_python_script(
+    """
+SOLN = STAT_STRUC.Solution
+
+TOT_DEF1 = SOLN.AddTotalDeformation()
+TOT_DEF1.DisplayTime = Quantity("680 [s]")
+
+EQV_STRS1 = SOLN.AddEquivalentStress()
+EQV_STRS1.DisplayTime = Quantity("680 [s]")
+
+EQV_PLAS_STRN1 = SOLN.AddEquivalentPlasticStrain()
+EQV_PLAS_STRN1.DisplayTime = Quantity("680 [s]")
+
+THERM_STRN1 = SOLN.AddThermalStrain()
+THERM_STRN1.DisplayTime = Quantity("680 [s]")
+
+# Solve Nonlinear Static Simulation
+
+#SOLN.Solve(True)
```

Comment:
```suggestion
SOLN.Solve(True)
```
Uncomment this once pyfluent script is ready